src/Common/src/Interop/Windows/mincore/Interop.CloseHandle.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Handle, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
Though I see that there is (or at least used to be) an FxCop rule that warns if you don't use MarshalAs on all Booleans in P/Invokes.  If that rule is still around, I guess we should leave this in.